packages/@aws-cdk-testing/framework-integ/test/aws-codebuild/test/integ.project-amazonlinux-image.lit.ts

```diff
@@ -0,0 +1,30 @@
+import { App, Stack } from 'aws-cdk-lib';
+import { Construct } from 'constructs';
+import { BuildSpec, LinuxBuildImage, Project } from 'aws-cdk-lib/aws-codebuild';
+import { IntegTest } from '@aws-cdk/integ-tests-alpha';
+
+class AmazonLinuxImageTestStack extends Stack {
+  constructor(scope: Construct, id: string) {
+    super(scope, id);
+
+    /// !show
```

Comment:
Hey, could you also remove: `/// !show` and `/// !hide`.

Reply:
fixed this too 😄